charts/spire/templates/agent-daemonset.yaml

```diff
@@ -13,6 +13,10 @@ spec:
     type: RollingUpdate
   template:
     metadata:
+      {{- with .Values.agent.service.annotations }}
```

Comment:
Could we do this change as a separate PR to patch the 0.7.x version?

Reply:
Makes sense, are you okay will all the non-UpstreamAuthority related changes being in same patch version?